### ISSUE TITLE
add explanation for event types

### ISSUE THREAD

**Issue by mathias-gibson** [codeforamerica/mdc-step-wizard#23]
![pasted_image_6_26_15__3_23_pm](https://cloud.githubusercontent.com/assets/8147336/8388573/8e70d3d0-1c18-11e5-93c7-e00741e1449e.png)

**Comment by phiden** [comment]
Yes, but we're not capable of defining these things -- I think they're legal-ish definitions, and the three of us don't have the expertise -- we need someone within the county to do this. @mathias-gibson can you see if you can chase these definitions down? cc @ErnieAtLYD in case you have bandwidth to handle this.
